bin/add-idam-clients.sh

```diff
@@ -14,4 +14,6 @@ ${dir}/utils/idam-create-service.sh "am_role_assignment" "am_role_assignment" "a
 
 ${dir}/utils/idam-create-service.sh "ccd_data_store_api" "ccd_data_store_api" "idam_data_store_client_secret" "http://ccd-data-store-api/oauth2redirect" "false" "profile openid roles manage-user"
 
-${dir}/utils/idam-create-service.sh "ccd_case_disposer" "ccd_case_disposer" "idam_ccd_case_disposer_secret" "http://ccd-case-disposer/oauth2redirect" "false" "profile openid roles"
\ No newline at end of file
+${dir}/utils/idam-create-service.sh "ccd_case_disposer" "ccd_case_disposer" "idam_ccd_case_disposer_secret" "http://ccd-case-disposer/oauth2redirect" "false" "profile openid roles"
+
+${dir}/utils/idam-create-service.sh "ccd_next_hearing_date_updater" "ccd_next_hearing_date_updater" "AAAAAAAAAAAAAAAA" "http://ccd-next-hearing-day-updater-service/oauth2redirect" "false" "profile openid roles"
```

Comment:
The `redirect_uri` value has been changed in [application.yml](https://github.com/hmcts/ccd-next-hearing-date-updater/blob/master/src/main/resources/application.yaml#L22) can you update the value used in this command to match.

Reply:
Or update this and equivilant in hmcts/ccd-next-hearing-date-updater#40 not to use the word day or service in the URI: "http://ccd-next-hearing-day-updater-service/oauth2redirect"

Reply:
resolved in d56fc0ef1ac9570b245f7074bfc3116d01f131b8